doc/CMakeLists.txt

```diff
@@ -85,7 +85,10 @@ if(DOXYGEN_FOUND)
     set(DOXYGEN_BUILTIN_STL_SUPPORT YES)
     set(DOXYGEN_USE_MDFILE_AS_MAINPAGE "../README.md")
     set(DOXYGEN_EXCLUDE_PATTERNS "*/node_modules/*")
-    set(DOXYGEN_IMAGE_PATH "./")
+
+    set(DOXYGEN_IMAGE_PATH "")
```

Comment:
Is there a reason to set it to an empty string before appending?

Reply:
Nope! The C++ developer in me feels like that's an uninitialized read, but I checked the cmake docs and `list(APPEND)` treats a non-existent list is empty. I'll change it.